doc/FUZZING.md

```diff
@@ -0,0 +1,59 @@
+# Fuzz testing
+
+C-lightning currently supports coverage-guided fuzz testing using [LLVM's libfuzzer](https://www.llvm.org/docs/LibFuzzer.html)
+when built with clang.
+
+
+## Build the fuzz targets
+
+In order to build the C-lightning binaries with code coverage you will need a recent
+[clang](http://clang.llvm.org/). The younger the compiler version the better.
```

Comment:
what is a "younger compiler version"?

Reply:
A frenchism i guess: i meant it by opposition to older version :)

Reply:
```suggestion
[clang](http://clang.llvm.org/). The more recent the compiler version the better.
```